<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Currency</class>
 <widget class="QWidget" name="Currency">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label1">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>中国-元</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelCHN">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>美国-美元</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelUSA">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="1">
      <widget class="QPushButton" name="btnNum0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 28 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnNum9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 25 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnNum3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 28 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnNum5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 28 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnNum7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 28 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnDel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 25 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnNum8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 28 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnNum2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 28 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnPeriod">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 28 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnNum4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 28 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnNum1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 28 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnNum6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnClearAll">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 28 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
